ioxd_querier/src/lib.rs

```diff
@@ -160,13 +160,18 @@ pub struct QuerierServerTypeArgs<'a> {
     pub exec: Arc<Executor>,
     pub time_provider: Arc<dyn TimeProvider>,
     pub querier_config: QuerierConfig,
-    pub authz: Option<Arc<dyn Authorizer>>,
 }
 
 #[derive(Debug, Error)]
 pub enum Error {
     #[error("querier error: {0}")]
     Querier(#[from] querier::QuerierDatabaseError),
+
+    #[error("Authz configuration error for '{addr}': '{source}'")]
```

Comment:
This is a minor thing, but for the sake of consistency:

```suggestion
    #[error("authz configuration error for '{addr}': '{source}'")]
```

Errors should be lowercase, the reasoning being so that they compose together nicely:

```
error: calling something said: Problem loading data
```

vs:

```
error: calling something said: problem loading data
```

Likewise they shouldn't end in `.` (not that it does here!)